qcodes/instrument_drivers/Keysight/keysightb1500/message_builder.py

```diff
@@ -1203,6 +1203,20 @@ def clcorr(self,
                chnum: Union[constants.ChNr, int],
                mode: Union[constants.CLCORR.Mode, int]
                ) -> 'MessageBuilder':
+        """
+         This query command disables the open/short/load correction function
```

Comment:
one extra space of indentation here
```suggestion
        This query command disables the open/short/load correction function
```